packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,90 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
+  /*
+   #5693
+   missing finalized
+   */
+  {id: "getStateRoot", required: ["finalized"]},
+  {id: "getStateFork", required: ["finalized"]},
+  {id: "getStateFinalityCheckpoints", required: ["finalized"]},
+  {id: "getStateValidators", required: ["finalized"]},
+  {id: "getStateValidator", required: ["finalized"]},
+  {id: "getStateValidatorBalances", required: ["finalized"]},
+  {id: "getEpochCommittees", required: ["finalized"]},
+  {id: "getEpochSyncCommittees", required: ["finalized"]},
+  {id: "getStateRandao", required: ["finalized"]},
+  {id: "getBlockHeaders", required: ["finalized"]},
+  {id: "getBlockHeader", required: ["finalized"]},
+  {id: "getBlockV2", required: ["finalized"]},
+  {id: "getBlockRoot", required: ["finalized"]},
+  {id: "getBlockAttestations", required: ["finalized"]},
+  {id: "getStateV2", required: ["finalized"]},
+
+  /* missing route */
+  /* #5694 */
+  {id: "getSyncCommitteeRewards"},
+  {id: "getBlockRewards"},
+  {id: "getAttestationsRewards"},
+  {id: "getDepositSnapshot"}, // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  {id: "getBlindedBlock"}, // #5699
+  {id: "getNextWithdrawals"}, // #5696
+  {id: "getDebugForkChoice"}, // #5700
+  /* #6080 */
+  {id: "getLightClientBootstrap"},
+  {id: "getLightClientUpdatesByRange"},
+  {id: "getLightClientFinalityUpdate"},
+  {id: "getLightClientOptimisticUpdate"},
+  {id: "getPoolBLSToExecutionChanges"},
+  {id: "submitPoolBLSToExecutionChange"},
+
+  /* 
+   #6168
+   /query/syncing_status - must be integer
+   */
+  {id: "getHealth"},
+
+  /* 
+   #4638 
+   /query - must have required property 'skip_randao_verification'
+   */
+  {id: "produceBlockV2"},
+  {id: "produceBlindedBlock"},
```

Comment:
It would be good if we don't have to skip the whole route just because of one invalid / missing query param

Reply:
Good point, will do something similar as what I did for the required response properties.